### DISCUSSION TITLE
Free style Libre NFC read time out

### DISCUSSION THREAD

**Discussion by nareshvr** [NightscoutFoundation/xDrip#2689] [Q&A]
Hi Team ,

I developing CGM integration in my Android app.
 Currently i am using Xdrio+ repo for testing in that getting error "NFC read time out"

can you please help this issue.
<img width="206" alt="MicrosoftTeams-image (1)" src="https://user-images.githubusercontent.com/5001677/220120684-fbf2353b-29c2-4512-928e-587c6317c0aa.png">

**Comment by PhilTU** [comment]
There's a setting "use faster multi-block reading method" in xDrip+ NFC settings that I think needs to be off (default is on). We can see there is no response to the command. But this is a little bit of a guess on my part.

**Reply by nareshvr** [reply]
Right now use faster multi-block reading method is off 

<img width="206" alt="image" src="https://user-images.githubusercontent.com/5001677/220272708-7eb55398-eef0-4006-bd17-0159c046900f.png">

**Reply by PhilTU** [reply]
technically both options are off, I would usually have the bottom one enabled (any tag)

**Comment by Navid200** [comment]
> I developing CGM integration in my Android app. 

That's what xDrip does.  Can you explain why you have decided to develop something else?
Is something wrong with xDrip?  if yes, why not open a PR to suggest a fix?
